src/controller_examples/rabbitmq_controller/proof/common.rs

```diff
@@ -30,6 +30,14 @@ pub open spec fn rabbitmq_reconcile_state(step: RabbitmqReconcileStep) -> Rabbit
     }
 }
 
+pub open spec fn get_reconcile_state(rabbitmq: RabbitmqClusterView, step: RabbitmqReconcileStep) -> StatePred<ClusterState> {
```

Comment:
The fn name is confusing. Is it just a wrapper around at_expected_reconcile_states or you plan to extend it later?

Reply:
Why confusing?